### ISSUE TITLE
Tid is relunctant to show targets?

### ISSUE THREAD

**Issue by dirteat** [Zaretto/fg-aircraft#182]
Hi there,
I've been "flying" the f-14b, devel branch, during a week or so on MP-servers, fg version is next (Feb 2023).

The radar seems to be "very capricious". Targets are visible on the ecm, relatively fast, after tens of second, may be a minute max. But it takes ages for them to show up on the tid! Sometimes, they just never show up, even following another aircraft during minutes, at same altitude, relatively close range around 10nm, and after having set everything for the antenna to beam his ass :)

For the usage, I actually read the git commit of Nikolai, plus a few refs on the f-14, and I got pretty much what "bars" and "elevation" do, as well as the time required to sweep the space. In spite of my tries to tune all that in a hopefully reasonable manner, the tid remains reluctant to show the targets.

May be I missed some thing, so any advice or debugging guidance would be welcome!
Cheers,
chris.

**Comment by Zaretto** [comment]
@NikolaiVChr can you investigate this please.

Maybe we need to add some sort of debug dump radar state (including switches and instrument displays) that would permit further investigation

**Comment by NikolaiVChr** [comment]
@dirteat did you read the wiki on this github about the radar to understand when it wont pick up targets?

For example is there a chance the aircraft was notching?

Or the closure-rate too low to be detected in the mode you were in?

As for elevation, did you take into account the curvature of the the earth, the radar will do that, so a target at same altitude as you, but a distance out will be below 0 elevation.

And lastly, if something is off, what aircraft did you test this on? Is important in case there is an issue in the RCS system.

**Comment by dirteat** [comment]
Yes I did read the wiki, still I do not exclude bad usage, I am far from being an expert on the f-14b RIO. But even in sort-of-easy situations, targets are very rarely showing up.

For instance, target heading towards me from 30nm during a few minutes, waiting for a merge, nothing, even at merging time. I am getting visual contact before the radar! For these situations, I had time to try  different elevations, bars, and modes, and nothing shows up.  Following another aircraft, same symptoms (then possibly, the closure rate was too low).
 
Very rarely, targets show up, usually in bunches, I mean more than one suddenly appear on the tid, then I can SST them, untag, all seems to work, but soon after they're gone. I am not dogfighting on OPRF (yet) so I don't thing so my flying mates were notching :)

What I've observed to work fine, is the "d" (pal mode). In the few dogfights I had with the f-14b, I was getting targets only  in PAL mode, while flying and escaping in all possible orientations, which were clearly not radar-friendly :)

I guess if there is a way to pop messages out of the terminal, I could see what's going on!

PS: Anyway, this new radar is very cool, we can "feel" to work of the RIO...
